sdk/communication/communication-phone-numbers/samples-dev/siprouting/manageSipRoutingConfiguration.ts

```diff
@@ -64,9 +73,22 @@ export async function main() {
     console.log(`Route ${route.name} with pattern ${route.numberPattern}`);
     console.log(`Route's trunks: ${route.trunks?.join()}`);
   }
+
+  // Clear configuration
+  await client.setRoutes([]);
+  await client.setTrunks([]);
 }
 
 main().catch((error) => {
   console.log("The sample encountered an error:", error);
   process.exit(1);
 });
+
+function getUniqueFqdn() {
+  const length = 12;
+  let random = 0;
+  do {
+    random = Math.floor(Math.random() * 10 ** length);
```

Comment:
I'm not a fan of obtaining a random number in a potentially very long while loop. I would do the following:

// Create leading digit which is 1 to 9.
int leadingDigit = Math.floor(Math.random() * 9) + 1;
int remainingDigits = Math.floor(Math.random() * 10 ** 11);
int fqdnNumber = leadingDigit * 10 ** 11 + remainingDigits;
return `sample.${fqdnNumber}.com`;

Reply:
Hello, thank you for another possible approach. In the end I used the solution with UUIDs.
const firstTrunkFqdn = `sample.${uuid()}.com`;